core/pom.xml

```diff
@@ -34,6 +34,10 @@
       <artifactId>galley-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.commonjava.maven.galley</groupId>
```

Comment:
Well, this is only for using constant HttpExchangeMetadata.FILE_EXTENSION - ".http-metadata.json". I was not sure when I added the dep for such a minor convenience. Where else can we put this constant (maybe in api? or we just use the literal ".http-metadata.json")